app/src/main/java/in/ac/iitb/gymkhana/iitbapp/MainActivity.java

```diff
@@ -47,6 +52,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
         NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
         navigationView.setNavigationItemSelectedListener(this);
+        if (ContextCompat.checkSelfPermission(this, android.Manifest.permission.ACCESS_FINE_LOCATION) !=
```

Comment:
You need to override `onPermissionsResult()`. Check the code in [Tum Tum Tracker](https://github.com/unstablebrainiac/Tum-Tum-Tracker/) to get an idea.